tools/dataset_converters/textdet/ilst_converter.py

```diff
@@ -155,7 +155,7 @@ def split_train_val_list(full_list, val_ratio):
     """Split list by val_ratio.
 
     Args:
-        full_list (list): List to be splited
+        full_list (list): List to be splitted
```

Comment:
Thanks for your contribution! But shouldn't it be "split"? :)

Reply:
@gaotongxiao 
Thank you for contacting me
Corrected the content to "split".